.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino
```

Comment:
EOF: 

To avoid this warning from github: 

![image](https://user-images.githubusercontent.com/10209180/175051559-1f8f3a55-1bde-4161-ba81-2c2d9326cc60.png) 

You should add an empty line at the end of the file (or linebreak), that line is called EOF. You can configure your visual studio code to add automatically the EOF: https://stackoverflow.com/a/44704969  

Next time that we see this, we will comment just EOF as a reminder

Reply:
Fixed, I already configured my VSC, thanks.

Reply:
Yes, I added .gitignore file